src/Fixture/FixtureCreator.php

```diff
@@ -358,25 +358,37 @@ public function setSymlinkAll(bool $symlink_all): void {
   }
 
   /**
-   * Creates a BLT project.
+   * Creates a project.
    */
-  private function createBltProject(): void {
-    $this->output->section('Creating BLT project');
+  private function createProject(): void {
+    $this->output->section('Creating project');
     $stability_flag = '--stability=alpha';
-    if (($this->isDev || ($this->sut && $this->sut->getPackageName() === 'acquia/blt'))) {
+    if (($this->isDev || ($this->sut && ($this->sut->getPackageName() === 'acquia/blt' || $this->sut->getPackageName() === 'acquia/drupal-recommended-project')))) {
       $stability_flag = '--stability=dev';
     }
+    // Handle D8 fixtures (default).
     $version = ($this->isDev)
       ? $this->blt->getVersionDev($this->drupalCoreVersion)
       : $this->blt->getVersionRecommended($this->drupalCoreVersion);
+    $project_template = 'acquia/blt-project';
+    // Handle D9 differently, since BLT Project no longer exists.
+    if (version_compare($this->getResolvedDrupalCoreVersion(), 9, '>=')) {
+      $project_template = 'acquia/drupal-recommended-project';
+      $version = ($this->isDev) ? 'dev-master' : '^1';
```

Comment:
If you want to not hardcode the project template name and branch versions, maybe they could exist in the `package.yml` config, along with some sort of parameter to indicate that they are fixture templates and _not_ full SUTs.

Reply:
I'm currently in the process of trying to use an alternative project template over in https://github.com/acquia/acquia_migrate/pull/188. We cannot use `acquia/drupal-recommended-project` on its own (for reason I'd be happy to explain if you must be convinced 😉). I would _love_ to have the ability to override this in a supported way. Right now, I'm doing the MVP/hackiest-thing-imaginable of [committing a `.patch` and applying that patch to ORCA in my `.travis.yml`](https://github.com/acquia/acquia_migrate/blob/deprojectify/.config/orca/override_orca_project_template.patch).

Reply:
From an engineering standpoint, I agree it would be elegant to allow the fixture to be selectable (even if just between `acquia/drupal-minimal-project` and `acquia/drupal-recommended-project`). But from a mission standpoint, I think this violates the first rule of ORCA, which is to replicate a single prescribed user journey and ensure all company packages work together. The more configurability ORCA adds, the less harmonized the testing process becomes.

The decision isn't really mine to make, but I think I'd prefer to fix whatever compatibility issues exist between `acquia/drupal-recommended-project` and acquia_migrate.

Reply:
The thing is that the migrate product is itself a prescribed user journey. It begins in the cloud UI and ends in a working Drupal 9 site. Along the way, components of the migrate product (not necessarily the `acquia_migrate` module) _generate_ a project level `composer.json` which is derived from `drupal/drupal-minimal-project` and has a suite of additional packages pre-`require`'d that are based on each individual customer's existing Drupal 7 site.

The reason I would like to create a custom project template is to exercise the code that generates that `composer.json` and ensure it results in an installable site. FWIW, it was @TravisCarden that suggested I explore creating a project template and looking at BLT for an example of how to do so.

Reply:
Ah yeah, well if the user journey is intentionally bifurcating and the only difference is the starting point (fixture base / project template) then yeah I guess ORCA needs to support that. Maybe you could still use acquia/drupal-minimal-project as a base.